test/e2e/windows/hybrid_network.go

```diff
@@ -79,6 +72,20 @@ var _ = SIGDescribe("Hybrid cluster network", func() {
 
 		})
 
+		ginkgo.It("should provide Internet connection for containers using DNS [Feature:Networking-DNS][LinuxOnly]", func() {
+			e2eskipper.SkipIfNodeOSDistroIs("windows")
+			ginkgo.By("checking connectivity to 8.8.8.8 53 (google.com) from Linux")
+			framework.ExpectNoError(
+				network.CheckConnectivityToHost(f, "", "connectivity-test", "google.com", 53, 30))
+		})
+
+		ginkgo.It("should provide Internet connection for containers using DNS [Feature:Networking-DNS][WindowsOnly]", func() {
+			e2eskipper.SkipIfNodeOSDistroIs("linux")
+			ginkgo.By("checking connectivity to 8.8.8.8 53 (google.com) from Windows")
+			framework.ExpectNoError(
+				network.CheckConnectivityToHost(f, "", "connectivity-test", "google.com", 53, 30))
```

Comment:
```suggestion
			assertConsistentConnectivity(f, windowsPod.ObjectMeta.Name, windowsOS, windowsCheck(linuxPod.Status.PodIP))
```

Reply:
@aojea how `windowsCheck(linuxPod.Status.PodIP)` can check connectivity to google.com, unable to understand.

Reply:
wrong copy paste , too the onefrom the linux pod instead